sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -547,8 +549,8 @@ def list_certificate_versions(self, name: str, **kwargs: "**Any") -> AsyncIterab
 
         Example:
             .. literalinclude:: ../tests/test_examples_certificates_async.py
-                :start-after: [START list_certificate_versions]
-                :end-before: [END list_certificate_versions]
+                :start-after: [START list_properties_of_certificate_versions]
```

Comment:
Some sample links are unchanged, do you want to rename them as well? (actual question, not passive aggressive statement 😁)

Reply:
changed those, thanks!